websites/R/Royal Road/presence.ts

```diff
@@ -0,0 +1,297 @@
+const presence = new Presence({
+		clientId: "1320791530956652544",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://i.imgur.com/7LzTPVj.png",
+}
+
+// Todo - Multilanguage functionality as hard to test atm due to bugs.
+async function getStrings() {
+	return presence.getStrings(
+		{
+			home: "general.viewHome",
+			search: "general.searchFor",
+			browse: "general.browsing",
+			buttonViewPage: "general.buttonViewPage",
+			reading: "general.reading",
+			chapter: "general.chapter",
+			buttonViewProfile: "general.buttonViewProfile",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
```

Comment:
```suggestion
		}
```
Passing the language is deprecated